### PR TITLE
Added Value Stream Mapping and DevOps Kaizen modules

### PR DESCRIPTION
**Pull request by tom-halpin**

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -13,8 +13,10 @@ DevOps
 dojo
 e.g.
 i.e.
+Kaizen
 Katacoda
 nvie
+VSM
```
Comment (adamdmharvey) [inline .spelling:19]: All capitals should not be necessary, as it is an acronym and should be ignored by default.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ To learn more about the Online DevOps Dojo and the Pet Clinic multi-verse in whi
 
 * The pull request / feature branch version control model.
 
+* The team deciding in an incident room strewn with coke cans, pizza boxes and stale air t to investigate the use of value stream mapping to better understand their current and desired processes.
```
Comment (adamdmharvey) [inline README.md:28]: typo (stale air t to) , and sentence may need a bit of rewording?

File: blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Following the positive community reaction to the launch of the Online DevOps Dojo, we are pleased to announce we have added two more modules to the dojo
```
Comment (adamdmharvey) [inline blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md:1]: Should end in either a period or maybe a colon to introduce bullets if that's the intent

File: blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* Value Stream Mapping
+* DevOps Kaizen
+
+These are in addition to the 5 modules which were included in the initial launch:
+
+* Welcome and Setup
+* Leading Change
+* Version Control
+* Continuous Integration
+* Shift Security Left
```
Comment (adamdmharvey) [inline blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md:12]:
```suggestion
* Shift Left on Security
```

The module is formally called this, so suggest for alignment

File: blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## VSM and DevOps Kaizen Modules
+
+Value Stream Mapping is a technique we encourage teams to understand the current state and to document the desired state of their process aka their value stream.
```
Comment (adamdmharvey) [inline blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md:16]: The initial wording seems a bit off.  Is the intent a) we encourage teams to use the technique and b) the technique does XYZ?  if so may need a bit more to it, but happy to collaborate on it - can suggest something if it helps

**Review by adamdmharvey — CHANGES_REQUESTED** [review]
Excited about this! Some initial feedback.

File: .spelling (hunk 1 above)
Comment (tom-halpin) [reply .spelling:19]: changed

File: README.md (hunk 2 above)
Comment (tom-halpin) [reply README.md:28]: done

File: blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md (hunk 5 above)
Comment (tom-halpin) [reply blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md]: Done

**Comment by tom-halpin** [comment]
@adamdmharvey thanks for the review, changes made / suggestions implemented.

**Review by pberthonneau — CHANGES_REQUESTED** [review]
Looks good to me except the 2 minor suggested changes about order values

**Comment by tom-halpin** [comment]
@pberthonneau good catches, requested changes made.

**Review by pberthonneau — APPROVED** [review]

**Comment by tom-halpin** [comment]
Merged based on having one approved PR and having made the changes requested in the other.

If more changes, corrections or improvements are needed they can be addressed via a subsequent a PR

**Pull request merged**